src/Google.Api.Gax/ServiceSettingsBase.cs

```diff
@@ -19,6 +20,19 @@ namespace Google.Api.Gax
     /// </remarks>
     public abstract class ServiceSettingsBase
     {
+        protected ServiceSettingsBase()
+        {
+            UserAgent = new UserAgentBuilder()
+                .AppendDotNetEnvironment()
+                .AppendAssemblyVersion("gax", typeof(CallSettings))
```

Comment:
Just thinking - I wonder if this should be gax-dotnet. Might want to check with Omar... ditto grpc